### ISSUE TITLE
not enough test 

### ISSUE THREAD

**Issue by Heathercoraje** [facn2/ApolloXI#68]
as @mattlub mentioned, test inside test.js is not testing any of functions. Also, I would suggest add more tests?

**Comment by veredrec** [comment]
@Heathercoraje good point. We put it there as a "placeholder" but didn't have time to get back to it and actually create the tests. And we should definitely add more tests...!

**Comment by rinoma** [comment]
@Heathercoraje 
close please

**Comment by Heathercoraje** [comment]
Have you add more tests??

**Comment by veredrec** [comment]
@Heathercoraje no, unfortunately we didn't have time to add it before our presentations.
